code/web/services/API/ListAPI.php

```diff
@@ -199,6 +200,7 @@ function getSearchableLists() {
 				if ($success) {
 					$row = $stmt->fetch();
 					$numTitles = $row['numTitles'];
+					// $displayListAuthor = $list['displayListAuthor'];
```

Comment:
Commented here.. should this be here at all?

Reply:
It should not, I will remove it.